js/ui/sessionMode.js

```diff
@@ -109,6 +109,40 @@ const _modes = {
             right: ['a11y', 'keyboard', 'aggregateMenu', 'userMenu',
                     'dateMenu', 'hotCorner']
         }
+    },
+
+    'user-coding': {
```

Comment:
Let's keep it like this for now and open another ticket to merge it back -- but you're still missing the backport of the code I mention above that makes use of this new session mode.